### ISSUE TITLE
DatedJourneyGroup: trainNumber --> vehicleNumber

### ISSUE THREAD

**Issue by schlpbch** [VDVde/OJP#173]
A vehicle journey doesn't need to be a train (e.g. bus, tram,...), does the `trainNumber` should be renamed to `vehicleNumber`.

# Proposal

`trainNumber` --> `vehicleNumber`

**Comment by skinkie** [comment]
Agreed if this is the meaning. But keep in mind that the meaning of trainNumber relates to a trainPart/Coach which for reservations and planning is likely something that should stick around.

**Comment by schlpbch** [comment]
Related to that: we additionally see the need for a line a number e.g. "S2" in local train.

Thus either "vehicleNumber" xor "lineNumber" need to be set.

**Comment by Aurige** [comment]
There are several discussions around this these days, involving UIC, Transmodel, ERA... we probably need to wait for the outcomes of these discussions and apply decided changes (if there are some) in the 2022 NeTEx revision.

**Comment by schlpbch** [comment]
![image](https://user-images.githubusercontent.com/18540761/151207395-afbbf657-413c-4371-8e6b-9e36618bf3a1.png)

**Comment by schlpbch** [comment]
The number "9710" obviously isn't a train number.

**Comment by Aurige** [comment]
But isn't this the name/shortname of the servicePattern (or event part of the DestinationDisplay) ?

The trainNumber is quite specific and may not be stable during a ServicePattern or even a ServiceJourney (can be associated to JourneyPart)

**Comment by ue71603** [comment]
add a VehicleNumber

**Comment by ue71603** [comment]
After discussion with @Aurige  I will create a pull request adding a VehicleNumber, but keeping TrainNumber.

**Comment by ue71603** [comment]
will be treated in pull request
